lib/vitalsigns/command.rb

```diff
@@ -2,15 +2,25 @@
 require 'json'
 
 module Vitalsigns
+  # Base class for commands
   class Command
-    def execute(command, options={})
-      Open3.popen3(command) do |stdin, stdout, stderr, wait_thr|
-        return stdout.read
+    def execute(command, _options = {})
+      Open3.popen3(command) do |_stdin, stdout, _stderr, _wait_thr|
+        return parse(stdout)
       end
     end
 
     def parse(output)
-      puts output
+      output.read
     end
   end
-end
\ No newline at end of file
+
+  def self.find_device(dire_path = '/var')
```

Comment:
`Vital::DeviceLookup.default` sounds more relevant to me.
`Vital.find_device` defeats the purpose of library scoping.

Another way to design this would be creating a singleton class and include such helper methods in it. So that it won't be initialized more than once per run.